content/copilot/github-copilot-chat/using-github-copilot-chat.md

```diff
@@ -106,9 +106,9 @@ To share feedback about {% data variables.product.prodname_copilot_chat %}, you
 ## Further reading
 
 - [AUTOTITLE](/free-pro-team@latest/site-policy/github-terms/github-copilot-pre-release-terms)
```

Comment:
```suggestion
- "[AUTOTITLE](/free-pro-team@latest/site-policy/github-terms/github-copilot-pre-release-terms)"
```
Let's put quotes around here because it is an internal article link